data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
Ehm I wrote that becuase of what you said, just read above haha

Not sure what you mean but imo a datapack should be set to a version ex 1.3 and you should have to use 1.3 for that datapack(talking about stable releases, latest atm is 1.2)
If we wanna get compat we should not do any new Lua changes, revert everything we have moved to Lua, updates to XML files etc that way you can use datapacks from 1.1 to 1.3.
If we revert metatags we can include 0.x aswell as 1.0.

Is that what you want?
A datapack should have a version: 1.2
If you wanna use 1.3 you have to update the files that we changed during 1.3 or stay with 1.2.